templates/manage/priorities.tpl.html

```diff
@@ -38,7 +38,43 @@
         $('#priority_form').submit(validateForm);
     });
     //-->
+
+	// Selectmenu – select icon of priority
+	$(function() {
+		$.widget( "custom.iconselectmenu", $.ui.selectmenu, {
+			_renderItem: function( ul, item ) {
+				var li = $( "<li>", { text: item.label } );
+				if ( item.disabled ) {
+					li.addClass( "ui-state-disabled" );
+				}
+				$( "<span>", {
+					style: item.element.attr( "data-style" ),
+					"class": "ui-icon " + item.element.attr( "data-class" )
+				})
+					.appendTo( li );
+				return li.appendTo( ul );
+			}
+		});
+
+		$( "#icons" )
+			.iconselectmenu()
+			.iconselectmenu( "menuWidget")
+				.addClass( "ui-menu-icons icon" );
+	});
 </script>
+<style>
+	/* Selectmenu – select icon of priority */
+	select {
+		width: 82px;
+	}
+	option.icon {
+		background-repeat: no-repeat !important;
+		padding-left: 20px;
+	}
+	.icon .ui-icon {
+		background-position: left top;
+	}
```

Comment:
do not inline css. add to global css file instead. page.css should have page specific css selectors